web/src/components/Schedule/Schedule.jsx

```diff
@@ -151,9 +161,21 @@ export default function Schedule() {
     });
   }
 
+  async function incorrectDates() {
+    setModalTitle("Start date after end date");
+    setModalDescription(
+      "It looks like your start date was set to a time after your end date. Fix and try again"
+    );
+    setShowModal(true);
+  }
+
   //IMPLEMENT CREATE/EDIT/DELETE EVENT HERE
   const commitChanges = ({ added, changed, deleted }) => {
     if (added) {
+      if (added.startDate > added.endDate) {
```

Comment:
Maybe we could add the extra logic that checks if startTime > endTime here (if on same date)? 

To avoid this: 
<img width="437" alt="image" src="https://user-images.githubusercontent.com/68136998/221943078-cf50158d-07b1-45d4-bdc4-5289161f2121.png">

Reply:
I'm pretty sure that it handles it as well, like the startDate actually handles the time. But will double check